utils/build-script

```diff
@@ -219,7 +219,11 @@ class BuildScriptInvocation(object):
         android_tgts = [tgt for tgt in args.stdlib_deployment_targets
                         if StdlibDeploymentTarget.Android.contains(tgt)]
         if not args.android and len(android_tgts) > 0:
-            args.android = True
+            # If building natively on an Android host, avoid the NDK
+            # cross-compilation configuration.
+            if not StdlibDeploymentTarget.Android.contains( \
+                       StdlibDeploymentTarget.host_target().name):
```

Comment:
This invocation simply checks if the build-script is running on an Android host by utilizing [`host_target()`'s `if 'ANDROID_DATA' in os.environ`](https://github.com/apple/swift/blob/90c478f6fc641037ce2eb1e61fe76a07f19cfd7e/utils/swift_build_support/swift_build_support/targets.py#L189), but I can just look for that Android environment variable directly here if that's more clear.

Reply:
Will do.

Reply:
Never mind, changed my mind after reading the longer comment I wrote, as this comment really covers them both. `android` and `build_android` both simply control the NDK configuration, and this check makes sure both are turned off on an Android host.